### ISSUE TITLE
Response Format with newest yii

### ISSUE THREAD

**Issue by balrok** [assayer-pro/yii2-sitemap-module#10]
I use the master branch of yii2 and get an invalid response format error with the robotsTxt action

Inside DefaultController use:
```php
Yii::$app->response->format = \yii\web\Response::FORMAT_RAW;
```
instead of
```php
Yii::$app->response->format = 'txt';
```

**Comment by serge-larin** [comment]
Sorry, it's my mistake. I use custom class ResponseFormatter and custom config for Yii::$app->components->response->formatters. But I miss this when create yii2-sitemap-module. I fix it as soon as posible.
You may use as temporary patch:

Add into your config 
```
'components' => [
...
        'response' => [
            'formatters' => [
                'txt' => [
                    'class' => '\app\base\ResponseFormatter',
                    'contentType' => 'text/plain',
                ],
            ],
...
],
```

Add file base/ResponseFormatter.php with class ResponseFormatter into project directory

```
<?php
namespace app\base;

use yii\base\Component;
use yii\web\ResponseFormatterInterface;

/**
 * TextPlainResponseFormatter formats the given data into an $contentType response content.
 *
 * It is used by [[Response]] to format response data.
 *
 */
class ResponseFormatter extends Component implements ResponseFormatterInterface
{
    /**
     * @var string the Content-Type header for the response
     */
    public $contentType = 'text/html';
    /**
     * Formats the specified response.
     * @param Response $response the response to be formatted.
     */
    public function format($response)
    {
        if (stripos($this->contentType, 'charset') === false) {
            $this->contentType .= '; charset=' . $response->charset;
        }
        $response->getHeaders()->set('Content-Type', $this->contentType);
        if ($response->data !== null) {
            $response->content = $response->data;
        }
    }
}
```

**Comment by serge-larin** [comment]
```
diff --git a/controllers/DefaultController.php b/controllers/DefaultController.php
index 6239f73..e705344 100644
--- a/controllers/DefaultController.php
+++ b/controllers/DefaultController.php
@@ -79,7 +79,8 @@ class DefaultController extends Controller
         $robotsTxt->sitemap = Yii::$app->urlManager->createAbsoluteUrl(
             empty($robotsTxt->sitemap) ? [$this->module->id.'/'.$this->id.'/index'] : $robotsTxt->sitemap
         );
-        Yii::$app->response->format = 'txt';
+        Yii::$app->response->format = Response::FORMAT_RAW;
+        Yii::$app->response->headers->add('Content-Type', 'text/plain');
         return $robotsTxt->render();
     }
 }
```

**Comment by serge-larin** [comment]
Fixed in version 1.3.4

**Comment by balrok** [comment]
Cool! Thank you 👍 :)
